<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LegendSettings</class>
 <widget class="QWidget" name="LegendSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>686</width>
    <height>381</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_5">
     <item row="0" column="0">
      <widget class="QGroupBox" name="legendGroupBox">
       <property name="title">
        <string>Legend</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="1" column="2">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Color</string>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="GuiUtils::ColorPushButton" name="legendColorButton">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="GuiUtils::FontPushButton" name="legendFontButton">
            <property name="text">
             <string>Font</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Font</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Position</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QComboBox" name="legendPositionCombo"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QGroupBox" name="legendItemGroupBox">
       <property name="title">
        <string>Legend Item</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <layout class="QGridLayout" name="gridLayout_7">
        <item row="2" column="0">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_6">
          <item row="1" column="1">
           <widget class="GuiUtils::FontPushButton" name="legendItemFontButton">
            <property name="text">
             <string>Font</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Font</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>Background Color</string>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="GuiUtils::ColorPushButton" name="legendItemBgColorButton">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>Border Color</string>
            </property>
           </widget>
          </item>
          <item row="2" column="3">
           <widget class="GuiUtils::ColorPushButton" name="borderColorButton">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string>Horizontal</string>
            </property>
           </widget>
          </item>
          <item row="3" column="3">
           <widget class="QComboBox" name="legendItemHorizontalCombo"/>
          </item>
          <item row="4" column="2">
           <widget class="QLabel" name="label_13">
            <property name="text">
             <string>Vertical</string>
            </property>
           </widget>
          </item>
          <item row="4" column="3">
           <widget class="QComboBox" name="legendItemVerticalCombo"/>
          </item>
          <item row="2" column="1">
           <widget class="GuiUtils::ColorPushButton" name="legendItemColorButton">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Font Color</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Column</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QSpinBox" name="legendItemColumnSpinBox"/>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Mode</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QComboBox" name="legendItemModeCombo"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item row="0" column="1" rowspan="2">
      <widget class="QGroupBox" name="groupBox_2">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="title">
        <string>Preview</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="1">
         <layout class="QVBoxLayout" name="demoLayout"/>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GuiUtils::ColorPushButton</class>
   <extends>QPushButton</extends>
   <header>comguiutils/colorpushbutton.h</header>
  </customwidget>
  <customwidget>
   <class>GuiUtils::FontPushButton</class>
   <extends>QPushButton</extends>
   <header>comguiutils/fontpushbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
